docs/concepts/case-study-snapshot.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Case study: Snapshot & IPFS'
+description: Explore how the leading decentralized gasless voting platform Snapshot uses IPFS as its main storage layer.
+---
+
+# Case study: Snapshot
+
+::: callout
+**"IPFS is the standard for storing data in a verifiable way that is fully transparent, thereby allowing us to build openly auditable governance systems"**
+
+_&mdash; Fabien, Founder and CEO, Snapshot Labs_
+:::
+
+## Overview
+
+::: right
+<img src="./images/case-studies/logo-snapshot.png" alt="Snapshot logo" width="200">
+:::
+[Snapshot](https://snapshot.org/#/) is an open-source **voting platform** for Web3 projects, DAOs, and communities that uses IPFS as its main storage layer.
+Snapshot works with projects deployed to Ethereum and EVM compatible chains and allows the use of governance tokens as a means to distribute voting power and eligibility.
+
+Snapshot is unique in its use of IPFS to store proposals and user votes using a technique known as ‘off-chain’ voting, where the cryptographic signatures proving user votes are persisted to IPFS instead of being stored on the blockchain.
```

Comment:
Do the signatures make it on-chain anywhere, or does this design rely purely on CID integrity?

Reply:
They don't.